### DISCUSSION TITLE
Preloading a JSX Component into Grapes Canvas

### DISCUSSION THREAD

**Discussion by kaimou1357** [GrapesJS/grapesjs#5692] [Q&A]
I've currently got some code generated in the form of JSX. It may or may not use other react libraries that are OUTSIDE the standard React code.

What is the proper way to import this? 

Looking at this [code sandbox, ](https://codesandbox.io/p/sandbox/grapesjs-jsx-setup-x07xf?file=%2Fsrc%2Findex.js%3A7%2C24), it's not really apparent to me HOW the code is actually imported when using React. If I decompile the code from JSX -> ES5, is it possible for Grapes to load ES5 code?

**Comment by artf** [comment]
I'd recommend checking [this discussion](https://github.com/GrapesJS/grapesjs/issues/1970) as the closest thing to React <> GrapesJS integration.
